stable/nfs-server-provisioner/values.yaml

```diff
@@ -53,6 +53,10 @@ storageClass:
   ## Ignored if storageClass.create is false
   name: nfs
 
+  ## StorageClass parameters
+  parameters:
+    mountOptions: vers=4.1
```

Comment:
this value should be commented or this is needed to run the chart?

Reply:
By default it will use nfs protocol version 4.2. This explicit option fixes the problem discussed here https://github.com/nfs-ganesha/nfs-ganesha/issues/192

Reply:
without this option all provisioned volumes are not writable

Reply:
thanks for the clarification!